docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -205,15 +212,15 @@ Standby daemons:
 . To migrate MDS to the right nodes, set the MDS affinity that manages the MDS failover:
```

Comment:
What are considered the “right nodes”? Are they the nodes that the user selected to be the target nodes?

Reply:
Yeah, +1 to change to `target nodes` and keep consistency.